Integrations/integration-Packetsled.yml

```diff
@@ -0,0 +1,1008 @@
33Kyk1f7WsCGFdCZoW7j1VTt2ai9T7pz4hCvKPozmcz5cAgD+ayka13j9fIYINbg+OFll2CHvtYsQdpiStN79hnAAQI21Wq1lY8xSvM+IUQw5xAw/+K7VfY4QfePcQ4YPHYS6t/cc5ylJsw8++OAGVxQAAAAAAAAgrMQYw1kAAAAAAGBMPPfcc9OSbkjJ1P5f/pP93w3+w0BfU+LUlj5OEqQt1PqqHjtJ0l6VMuMkDusNuXbb/cuNH75v7+ubWK4x1HlwbS9aa3Z7uDUkhe8Jt30G1n5N0sLJkydvcnUGgPpptVoNY8wrWdvtH5QPH2awDRq6BB1ir+2gvXxYw2XusH2LQhfhxvcJV4Sb0+c4B9q2JdMg4AAAAAAAAADEQ0UHAAAAAADGy4akqeqrJmTPQVWHtHFiVnUIt3a378dwX381fW32C1Gpwf8bnYdX2aHcN3BbHNNxDVSXAQDUQavVmjXGrJb77Iv3mWR3b+X6+Vn9Z3zR8RWv1eV8hehbzf1kyAoSfnP679NtWxNVHAAAAAAAAIDoqOgAAAAAAMCYePbZ55pJoqWev/Z3/jdiNYbBbX7jxKq8YH/sfuO4VnXwOycuawu59lBVHVzHKtu39/V1qYgQvkqCy7nLXutYVHZ46uTJk8tcpQGgHlqt1owx5qqkqbTt1Ycc7PbpX6Nru+/64gU5XNYaa/1257rcWOWqOcSYs1Q1hx1j1HjwwU+scyUBAAAAAAAA4qOiAwAAAAAAY+DZZ59rSFoaVjUGv+9RiF/Vwe3bYt3aio+l3PGHWlvZb/R1+7blcGO5fGOya6UEt7WH+BZni9lLDlLzyg6fbLVaDa7UADB8rVZr2hizriGGHHw+g20+M8OHHMLfM/h+noe9V/JdTzX3ln733eGroKW0XTBGM4QcAAAAAAAAgOpQ0QEAAAAAgBH37LPPzkr6TpjKBsOpvBBz7Emo6lB+ztjH6bcW/769r2/Maghhqzrkrdf9OMqtL8w+A8dw18mTJ3k4EACGaGtra0PSqbRtVYUcyoQDqg05hK38ULTeaqpO+FRV8BvLZf/0eWtTzWFHUuMTnyDgAAAAAAAAAFSNig4AAAAAAIywZ555dlpS30M3darqEKo6QriqDqHGsT1P5c5tyG/ktd2n7Lf0uj3oF7Zv+vkyhefB/5yGrwLh+gBn3R5EzXu/GWOarVZrlis3AAzH1tZWU4QcIrXb33O5fAFZ+KoT7veG8Sp2hb8Hj/B3gQuSZgg5AAAAAAAAAMNB0AEAAAAAgNG2LulY57d1CCm4zRvuQf1Q87nvEydcEWYd8R5M8zmPYed1KVJqExLwn7vuYQf/43Hbx2oNU8aYDcIOAFC9bshhKe2zofy1Pt7nkO9nY7Wf4fb3J67rLXtv5L/O2PeNIe8l/efM2WdH0l2f+MTHFz/xiU/c5AoCAAAAAAAADAdBBwAAAAAARtQzzzy7IulUnUIK9au8EHPsugQpys1Z7TfzhngYz6fKRPG3J9ftgchwYYfQVRp8vuW7z1S3ssM0V3EAqMbW1lZDGSGHeJ8l5T4fXT7DYwn1WR96nth9YwUaqgsC+wVje/bpVnH4OFUcAAAAAAAAgCFLhvUPxAAAAAAAwN8zzzyzIOkPpKTzF/yk76/7Tm19LcngNtu2wW3ua7FfX73GLj5PvvvEnDPE+LHn9eubtZ4kfZEe48VuL15zmWPxWZ/b2ouPS9K1JEkWTp48ybckA0BE3ZDDK4fbRyXk0OlX12oOoapSlFtTiPUXnb+y58I3XFF2/LxjTdlnR1Lj4x8n4AAAAAAAAADUBRUdAAAAAAAYMc8888y0pMgP4FDVwa4tfFWH2HOWHd/tOzNCVFFw6Vs0n381hGF9V8gYV3Y4bozZoLIDAMRDyKG6SkxFx2K3Xv/P9TL3WCHvCUfzntVI0pqkGUIOAAAAAAAAQL3cyikAAAAAAGC0GKP1JNHUobaBb063bfNcg+U4RoPVE8q01WPsUOe7f7v7Osu/xi7H5H5ObMYqXm9WX/dz02k3GdUQwq6pXLvt+9D1WPrPmesa3dZvdWzHjTHrkha4sgNAWFtbW7MqDDmY0tsmN+TgEh4os96s+xDbc2nfN0QAtfg43bf7jO9wrNvqVHHY4KoBAAAAAAAA1A8VHQAAAAAAGCFPP/3MiqRTBy3+lRfKVGMINc5wqzrEO/7ih9XCP7DlMqf/A3S2a7EZy+9hPP9qD739XcozDP9Bybz11q2yg/22fae2traaXN0BIJxuyGHD5fObkEPY9sK7iwD3CKGrTriss+x9Xpz7T+fyW09JZpaQAwAAAAAAAFBfidt/2AUAAAAAAMPy9NPPLEj6g/2/1O9/Q/rBV6WXaRvcdrDdti3UOKMwtt/5Lhqn7JxZ+4cf3+1YXeYNsZ7iNSWZCw01T9j15q05yS0FUTyfzxrDbdu3Njc31+BKDwDl9IQcBqp/9fxJNtvsgwx52yY75OCy5hDz+661eD8TtG+5Y/A7Bz37dKs4nN3gigEAAAAAAADUGxUdAAAAAAAYAU8//cy0pPX0rcOp6lBd5QU71VV1cD+3PhUWylZwcHt9wn1Dr++8/lUmjNfPTZlKCHWr7JD/RSahKzuUGy/n+Jao7AAA5aSFHNyq7fgGGcJ9RocKOcQWOuRQ7r6p7H1c+fvT8vfcJthaCo71SUmzhBwAAAAAAACA0XArpwAAAAAAgFFg1qVk4Jt5D38zepm2jHk1WPXAti3UOINt/esPtUbbdbuc22rm9H0tyq7f7Viz5s1/rbL6+o/Ru36TUQ3Bf/447cVrzj4W+/Nmv5Zy4+Uc39LW1pao7AAA7rJCDoOfByG3la8McfizzKU9926owqoNbsfiO0+IMInNmtyP2zNwYL0Wl7Fy9r8mqXH27NmrXC0AAAAAAACA0ZH4/AMxAAAAAACoztNPP70s6bfSHiLvf1A4Kd2WPnbi3Ja3vvqN7XP8fuc2/Tn0JMicdvvbnweXc2Y/r+s5Knse7NaUHRAIt9a
```

Comment:
I think you throw also the original error in case the response failed. The login request could for several reasons, not only `Authorization failed`, so we need more details